api/tests/unit/app_analytics/split_testing/test_unit_split_testing_tasks.py

```diff
@@ -0,0 +1,139 @@
+        type=STRING,
+        string_value="mv_feature_option2",
+    )
+    MultivariateFeatureOption.objects.create(
+        feature=feature2,
+        default_percentage_allocation=50,
+        type=STRING,
+        string_value="mv_feature_option3",
+    )
+    MultivariateFeatureOption.objects.create(
+        feature=feature2,
+        default_percentage_allocation=50,
+        type=STRING,
+        string_value="mv_feature_option4",
+    )
+
+    identity1 = Identity.objects.create(
+        identifier="test-identity-1", environment=environment
+    )
+    identity2 = Identity.objects.create(
+        identifier="test-identity-2", environment=environment
+    )
+    identity3 = Identity.objects.create(
+        identifier="test-identity-3", environment=environment
+    )
+    identity4 = Identity.objects.create(
+        identifier="test-identity-4", environment=environment
+    )
+
+    # Create evaluation for both features for identity1
+    FeatureEvaluationRaw.objects.create(
+        feature_name=feature1.name,
+        environment_id=environment.id,
+        evaluation_count=2,
+        identifier=identity1.identifier,
+    )
+    FeatureEvaluationRaw.objects.create(
+        feature_name=feature2.name,
+        environment_id=environment.id,
+        evaluation_count=1,
+        identifier=identity1.identifier,
+    )
+
+    # Create evaluation for only feature2 for identity2
+    FeatureEvaluationRaw.objects.create(
+        feature_name=feature2.name,
+        environment_id=environment.id,
+        evaluation_count=1,
+        identifier=identity2.identifier,
+    )
+
+    # Create evaluation for only feature2 for identity3
+    FeatureEvaluationRaw.objects.create(
+        feature_name=feature2.name,
+        environment_id=environment.id,
+        evaluation_count=1,
+        identifier=identity3.identifier,
+    )
+
+    # Create duplicate evaluations for only feature1 for identity4
+    FeatureEvaluationRaw.objects.create(
+        feature_name=feature1.name,
+        environment_id=environment.id,
+        evaluation_count=1,
+        identifier=identity4.identifier,
+    )
+    FeatureEvaluationRaw.objects.create(
+        feature_name=feature1.name,
+        environment_id=environment.id,
+        evaluation_count=1,
+        identifier=identity4.identifier,
+    )
+
+    # Create conversion events for identity3 and identity 4
+    ConversionEvent.objects.create(
+        environment=environment,
+        identity=identity3,
+    )
+
+    ConversionEvent.objects.create(
+        environment=environment,
+        identity=identity4,
+    )
+
+    # When
+    _update_split_tests()
+
+    # Then
+    split_tests = SplitTest.objects.all()
+    assert len(split_tests) == 4
+    assert sum([st.evaluation_count for st in split_tests]) == 5
+    assert sum([st.conversion_count for st in split_tests]) == 2
+    for st in split_tests:
+        assert st.pvalue == 1.0
+        assert st.statistic == 0.0
```

Comment:
These seem like very specific numbers, is there a reason we're getting these? Probably because the test has no statistical significance but I would have expected them to be (infinitesimally) close to 1 and 0, but probably not exactly those values.

Reply:
In a lot of statistics stuff you have cut offs for the lower numbers. It's not a smooth curve until a threshold has been reached.